invenio_theme/ext.py

```diff
@@ -76,3 +83,44 @@ def _generator_func_or_str():
     def icons(self):
         """Return icons."""
         return ThemeIcons(self.app.config["APP_THEME"], self.app.config["THEME_ICONS"])
+
+
+class InvenioApiTheme:
+    """Invenio theme extension for API app."""
+
+    def __init__(self, app=None):
+        r"""Extension initialization.
+
+        :param app: An instance of :class:`~flask.Flask`.
+        """
+        if app:
+            self.init_config(app)
```

Comment:
Didn't register it back on the app, since it doesn't really provide anything for later use. It's just to load configs.